src/client/components/TotalPriceCard/TotalPriceCard.styles.css

```diff
@@ -0,0 +1,28 @@
+.wrapper {
+  width: 18rem;
+  height: 10rem;
+  border: 1px solid lightgray;
+  border-radius: 0.5rem;
+}
+
+.wrapper:hover {
+  border: 1px solid black;
```

Comment:
I think we don't need border radius and hover for this component. This component is made as a rectangle around it in figma
```suggestion

```

Reply:
yes i will remove them. i just added for better presentation. :)